docs/AndroidBuildingFromSource.md

```diff
@@ -143,6 +143,18 @@ gradle.projectsLoaded {
 }
 ```
 
+## Building for Maven/Nexus deployment
+
+If you find that you need to push up a locally compiled React Native .aar and related files to a remote Nexus repository, you can.
+
+Start by following the `Point Gradle to your Android SDK` section of this page. Once you do this, assuming you have Gradle configured properly, you can then run the following command from the root of your React Native checkout to build and package all required files:
+
+```
+./gradlew ReactAndroid:installArchives
+```
+
+This will package everything that would typically be included in the `android` directory of your `node_modules/react-native/` installation in the root directory of your React Native checkout. 
```

Comment:
@MattFoley sorry, if you have a moment, what are the steps after these files are created? How can mere mortals upload a forked / custom version to maven or consume it any other way?